packages/block-library/src/image/edit.js

```diff
@@ -259,7 +259,7 @@ export function ImageEdit( {
 	}
 
 	function updateAlignment( nextAlign ) {
-		const extraUpdatedAttributes = isWideAligned
+		const extraUpdatedAttributes = [ 'wide', 'full' ].includes( nextAlign )
```

Comment:
The wide alignment issue was a different regression introduced here https://github.com/WordPress/gutenberg/commit/1063573c5f1c95f05b4ab8e42b763158c7c79cac#r39498180, I pushed a fix